src/AltinnCore/Common/Models/AltinnStudioViewModel.cs

```diff
@@ -18,7 +18,7 @@ public class AltinnStudioViewModel
         public string Org { get; set; }
 
         /// <summary>
-        /// Gets or sets the service
+        /// Gets or sets the service (repository name, same as "app"). The ViewModels will phase out, so no need to update to "app" here.
```

Comment:
not a fan of this change.. if no change is needed just leave it instead of commenting why you aren't changing it..

Reply:
This was not supposed to be commited, was not finished and was not going to be like this. Due to corrupt files in local repo I had to push all changes and do a completely new clone of project to continue working.